shuup_tests/front/test_checkout.py

```diff
@@ -72,3 +77,102 @@ def test_address_phase_anonymous_user(rf):
     resp = view_func(request)
     assert 'company' in resp.context_data['form'].form_defs
 
+
+# BROWSER TESTS BELOW
```

Comment:
Unneccessary notification about incoming browser tests. you should probably separate the browser tests for checkout in it's own file if you also think that this file needs to be divided.